src/components/appRouter.js

```diff
@@ -14,31 +14,6 @@ define(['loading', 'globalize', 'events', 'viewManager', 'layoutManager', 'skinM
         },
         showSettings: function () {
             show('/settings/settings.html');
-        },
```

Comment:
Well almost all function removals you see are redefined in site.js. I just removed because they were doing nothing. Then we have the handleBackToDefault method that i also removed. That function would only be called when authenticate was called. In my point of view, that doesn't make any sense making a back operation right after a authenticate operation. Why would i want to exit the app (This can lead to a exit for a tizen context) when making a login? doesn't make much sense to me, so that's why i remove the condition and invocation. If that proofs any future problem, then we deal with it.

I also made a refactoring to the controllers that are loaded. Webpack doens't like dynamic module resolution, but it supports partial dynamic resolution. We can achieve that by using string literals. It gives a base location for webpack to fetch all files into possible locations given the literal. So that solves the problem for full dynamic module resolution by webpack.

I think it's all what i changed in this file.